compiler/include/driver.h

```diff
@@ -260,10 +260,11 @@ extern int breakOnID;
 extern int breakOnRemoveID;
 
 extern int fGPUBlockSize;
-extern char fGpuArch[16];
+const int gpuArchNameLen = 16;
```

Comment:
Not that I'd hold up this PR over it but I could imagine this taking more than 16 characters at some point if we wanted to support passing multiple arches in (for example if we wanted the fatbin to target several different GPUs).

Anyway, we can cross that bridge when we come to it.

Reply:
Yeah, I agree.

Reply:
FWIW my own reaction to this kind of thing is "Why doesn't it use `std::string` anyway?"